container-search/pom.xml

```diff
@@ -60,6 +60,11 @@
       <artifactId>json</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
+      <scope>provided</scope>
+    </dependency>
```

Comment:
This should be removed again if it is unused.

Reply:
👍